django/core/management/base.py

```diff
@@ -58,8 +58,16 @@ def __init__(
     ):
         self.missing_args_message = missing_args_message
         self.called_from_command_line = called_from_command_line
-        if PY314 and not PY315:
-            kwargs.setdefault("suggest_on_error", True)
+        if PY314:
+            if not PY315:
+                kwargs.setdefault("suggest_on_error", True)
+            if (
+                os.environ.get("NO_COLOR")
```

Comment:
This environment variable hasn't been set yet; we also need to check for an incoming ``--no-color`` argument instead. Please add a test.